streams/src/main/java/org/apache/kafka/streams/kstream/internals/KTableImpl.java

```diff
@@ -332,7 +332,7 @@ private <VR> KTable<K, VR> doMapValues(final ValueMapperWithKey<? super K, ? sup
         final GraphNode tableNode = new TableProcessorNode<>(
             name,
             processorParameters,
-            storeFactory
+            null
```

Comment:
If we do not need the storeFactory, could we just remove this param?

Reply:
Yes, but I plan to do that once I've transitioned the other callers of `TableProcessorNode`

Reply:
actually @ableegoldman pointed out below that the other caller of this constructor (fk-join) has already been transitioned, so I've cleaned this up